### DISCUSSION TITLE
2 hours to push update on webpage?

### DISCUSSION THREAD

**Discussion by jbeemer** [community/community#22203] [Other Feature Feedback, Questions, & Ideas]
<p>I pushed several updates late this afternoon around 4:30 pm, and they didn’t update on the website until several hours later. It finally updated but for no apparent reason, i.e., nothing I did brought about that eventual update.</p>
<p>So I’m trying to figure out why this happened and hope to avoid it down the road. It was <strong>not</strong> a browser refresh issue. Is this something that could have originated with Github? In other words, is a 2 hour delay from push to update not unheard of with Github?</p>
<p>Thanks for your help!</p>

**Comment by jbeemer** [comment]
<p>Anyone? I’m really at a loss here on what the problem might be. I’m still getting well over an hour for updates to appear. Never had this issue before, and my last set of updates several weeks ago didn’t encounter this excessive time lag.</p>
<p>Thanks.</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/jbeemer">@jbeemer</a>, welcome to the GitHub Support Community! Pages sites can take up to 10 minutes to finish building and then a further 10 minutes to deploy across the CDNs, so it can take up to 20 minutes for your site to finish deploying.</p>
<p>If you’re still not seeing changes after this time then it could be related to local caching of your site in your browser, caching at the CDN level, or some other issue. Without being able to see your repository/site though it’s hard to tell exactly what’s going on.</p>

**Comment by jbeemer** [comment, marked as answer]
<p><a class="mention" href="/thomasshaped">@thomasshaped</a>, thank you for your reply - much appreciated! I think I’ve isolated the problem, which appears to be unrelated to Github. I’m told the slowdown is likely coming from the Travis-CI service.</p>
